app/controllers/categories_controller.rb

```diff
@@ -2,6 +2,10 @@ class CategoriesController < ApplicationController
 
   def index
     @categories = Category.all
+    respond_to do |format|
+      format.html
+      format.json { render json: @categories }
```

Comment:
Consider (in a future pull request) refactoring this out into its own API controller/route.